.github/workflows/backport.yaml

```diff
@@ -25,7 +25,3 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           auto_backport_label_prefix: backport-to-
-
```

Comment:
the action is working fine, but it fails when it tries to print the log. https://github.com/mesosphere/kommander-applications/runs/6562935492?check_suite_focus=true